src/clojure/nrepl/server.clj

```diff
@@ -10,8 +12,21 @@
    [nrepl.misc :refer [log response-for returning]]
    [nrepl.transport :as t])
   (:import
+   [java.io PipedInputStream PipedOutputStream]
    [java.net InetAddress InetSocketAddress ServerSocket Socket SocketException]))
 
+(defn- resolve-ns-symbol
+  "Resolve a namespaced symbol to a var. Returns the var or nil if
+  the argument is nil or not resolvable."
+  [tag var-sym]
```

Comment:
Btw, what do we need this tag for?

Reply:
Sorry, it was supposed to be generic to tell us what was trying to resolve if at fail, gonna remove it